views/index.erb

```diff
@@ -193,29 +201,33 @@
         <% end %>
       </div>
     </div>
-        <% if userinfo %>
-          <div class="userinfo">
-            <div class="p2 clearfix bg-white fg-black">
-              <span>Received user info:</span>
-                <% case userinfo.fetch('acr')
-                     when 'http://idmanagement.gov/ns/assurance/loa/3' %>
-                  <span>Login type: <strong>LOA3</strong></span>
-                  <span class="ml1 mr1">|</span>
-                  <span>SSN: <strong><%= maybe_redact_ssn(userinfo[:social_security_number]) %></strong></span>
-                <% when 'http://idmanagement.gov/ns/assurance/loa/1' %>
-                  <span>Login type: <strong>LOA1</strong></span>
-                <% else %>
-                  <span>Login type: <strong>Unexpected ACR: <%= userinfo.fetch('acr') %></strong></span>
-                <% end %>
-              <ul>
+    <% if userinfo %>
+      <div class="userinfo">
+        <div class="p2 clearfix bg-white fg-black">
+          Received user info:<br/>
+          <% case userinfo.fetch('acr')
+             when 'http://idmanagement.gov/ns/assurance/loa/3' %>
+            Login type: <strong>LOA3</strong><br/>
+            < class="ml1 mr1">|<br/>
```

Comment:
Looks like the tag name was removed, but not all of the markup. Should it have been kept?

```suggestion
            <span class="ml1 mr1">|</span><br/>
```

Or removed altogether?

```suggestion
```